package.json

```diff
@@ -75,8 +75,16 @@
     "through2": "^2.0.0",
     "typescript": "^3.6.3"
   },
+  "workspaces": [
+    "codemods/*",
+    "eslint/*",
+    "packages/*"
+  ],
   "resolutions": {
-    "@lerna/collect-updates": "https://github.com/babel/lerna.git#babel-collect-updates"
+    "@lerna/collect-updates": "https://github.com/babel/lerna.git#babel-collect-updates",
+    "@lerna/package": "patch:@lerna/package@npm:3.16.0#.yarn-patches/@lerna/package.patch",
+    "@lerna/package-graph": "patch:@lerna/package-graph@npm:3.18.5#.yarn-patches/@lerna/package-graph.patch",
+    "@lerna/pack-directory": "patch:@lerna/pack-directory@npm:3.16.4#.yarn-patches/@lerna/pack-directory.patch"
```

Comment:
This is a mess. In the near future (maybe in this PR) I will use `yarn patch` also for `@lerna/collect-updates` (instead of a fork).
Then, I will try to understand if it's easier to make lerna work with yarn 2 by default, or if it's better to re-implement the parts that we are using as a yarn plugin.